src/plugins/rust/README.md

```diff
@@ -37,14 +37,14 @@ syntax = { path = "syntax" }
 
 ### 3. Configure dependencies of the `syntax` crate
 
-_Syntax_ tool generates a parser which depends on two external crates: [regex](https://doc.rust-lang.org/regex/regex/index.html), and [lazy_static](https://crates.io/crates/lazy_static). Let's add them to our `syntax` library dependencies list.
+_Syntax_ tool generates a parser which depends on two external crates: [onig](https://docs.rs/onig/4.3.2/onig/), and [lazy_static](https://crates.io/crates/lazy_static). Let's add them to our `syntax` library dependencies list.
```

Comment:
Is `onig` considered a better replacement for `regex` create these days (haven't coded in Rust for a while)?

Reply:
Not sure about rust world, but in real widely-used applications – for sure, onig is battle tested and famous library.

> Oniguruma (鬼車) by K. Kosako is a BSD licensed regular expression library that supports a variety of character encodings. The Ruby programming language, in version 1.9, as well as PHP's multi-byte string module (since PHP5), use Oniguruma as their regular expression engine.[2] It is also used in products such as Atom,[3] GyazMail Take Command Console, Tera Term, TextMate, Sublime Text and SubEthaEdit.

From [wikipedia](https://en.wikipedia.org/wiki/Oniguruma)

Reply:
And in terms of functionality, `onig` > `regex`, because `onig` do support lookeahed/lookbehind techniques instead of `regex` crate

Reply:
Cool, thanks, sounds good. If `onig` is better than `regex`, we can use it.